samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/DeliveryBean.java

```diff
@@ -1859,41 +1911,45 @@ public String next_page()
     forGrade = false;
 
     if (this.actionMode == TAKE_ASSESSMENT
-        || this.actionMode == TAKE_ASSESSMENT_VIA_URL)
+            || this.actionMode == TAKE_ASSESSMENT_VIA_URL)
     {
       syncTimeElapsedWithServer();
-	
+
       SubmitToGradingActionListener listener =
-        new SubmitToGradingActionListener();
+              new SubmitToGradingActionListener();
       try {
-    	  listener.processAction(null);
+        listener.processAction(null);
       }
       catch (FinFormatException e) {
-		  log.debug(e.getMessage());
-		  return "takeAssessment";
-	  }
-	  catch (SaLengthException sae) {
-		  log.debug(sae.getMessage());
-		  return "takeAssessment";
-	  }
+        log.debug(e.getMessage());
+        return "takeAssessment";
+      }
+      catch (SaLengthException sae) {
+        log.debug(sae.getMessage());
+        return "takeAssessment";
+      }
     }
```

Comment:
See L1903. This is a creation of a new function goto_question() which shares similar functionality / structure to the next_page() function. I copied and pasted the next_page() function from wget-ing the raw file off github, hopefully this will fix the diff. Else, I'm not sure what else to do.